src/dsp_tools/models/input_error.py

```diff
@@ -0,0 +1,131 @@
+from dataclasses import dataclass
+from typing import Protocol
+
+# pylint: disable=too-few-public-methods
+
+
+separator = "\n\t"
+list_separator = "\n\t- "
```

Comment:
I tend to avoid tab stops where ever possible. How would you feel about replacing them with 2 or 4 spaces?

Reply:
I researched the best practice. And I am convinced 
https://stackoverflow.blog/2017/06/15/developers-use-spaces-make-money-use-tabs/
(I think it is about the tabs or spaces in code which is a non-issue here, but still funny)